roles/linux_preparation/tasks/recore.yml

```diff
@@ -52,58 +20,41 @@
     - libnss-mdns
     - byobu
     - htop
-    - unzip
```

Comment:
Uhh, maybe leave that util?

Reply:
It is mentioned twice.

Reply:
Oh, the review only shows it to me once.